src/artifacts/mod.rs

```diff
@@ -1049,6 +1049,7 @@ pub enum BytecodeHash {
     Ipfs,
     None,
     Bzzr1,
+    Keccak256,
```

Comment:
We do not support `Ipfs` and `Bzzr1` either, just in case this enum is for `zksolc` only.